.github/workflows/bitstream.yml

```diff
@@ -82,11 +82,21 @@ jobs:
         if: steps.strategy.outputs.bitstreamStrategy != 'cached'
         run: |
           . util/build_consts.sh
+
```

Comment:
If I understand correctly, this commit is all to do with the change to the new FuseSoc version. Again, maybe we need to squash with that commit so that (we think) everything works at each commit?